docs/source/external_tutorials/tutorial_stardist.ipynb

```diff
@@ -0,0 +1,517 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Nuclei segmentation using stardist\n",
+    "\n",
+    "This tutorial shows how we can easily use an external segmentation packages in the *Squidpy* framework. Tp use the stardist package, pip-install stardist with `pip install stardist` and additionally `pip install tensorflow` as this is a requirement of the stardist package. For details on the segmentation model, see: Schmidt U., et al. *Cell Detection with Star-Convex Polygons.* MICCAI 2018 Proceedings, Part II (265-273). doi: 10.1007/978-3-030-00934-2_30 or https://github.com/stardist/stardist for the GitHub repository.\n",
```

Comment:
could you also add 1 line describing stardist and linking to the paper there? Similar to the tangram tutorial: https://squidpy.readthedocs.io/en/latest/external_tutorials/tutorial_tangram.html
**Tangram** (<cite data-cite="tangram">Biancalani et al. (2020)</cite>, [code](https://github.com/broadinstitute/Tangram)). The reference goes in docs/source/references.bib I think